src/entities/plugin.ts

```diff
@@ -1,19 +1,25 @@
 // The plugin entity
 export interface PluginInterface {
-  plugin_id: number,
-  plugin_version: string,
-  plugin_name: string
+  plugin_id: number;
```

Comment:
This needs to be an object definition with commas. not semicolons


not sure how this is working